Source/WebCore/Modules/websockets/WebSocketExtensionDispatcher.cpp

```diff
@@ -95,8 +95,7 @@ bool WebSocketExtensionDispatcher::processHeaderValue(const String& headerValue)
     }
 
     const CString headerValueData = headerValue.utf8();
```

Comment:
At some point we need to replace `CString` with explicit `UTF8String` and `Latin1String` classes and remove the need for this kind of casting.